### DISCUSSION TITLE
Welcome to pnpm-turbo-monorepo-template Discussions!

### DISCUSSION THREAD

**Discussion by sonofmagic** [sonofmagic/monorepo-template#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xq52301nzdm** [comment]
在原生小程序开启skyline的情况下，每次对wxml页面进行任意变更都会重新触发编译，在webview的render模式下是正常的，变更的东西会“冷启动”的显示在开发者工具中
